platform/Cargo.toml

```diff
@@ -22,7 +22,8 @@ name = "host"
 path = "src/main.rs"
 
 [dependencies]
-roc_std = { git = "https://github.com/roc-lang/roc", rev = "7b1f2d2ac1f6e0b7a283cfbe932318f37a0a6861" }
+roc_std = { path = "src/main_glue/roc_std", features = ["std"] }
```

Comment:
Isn't it risky to copy roc_std to this repo @lukewilliamboswell?
Fixes and changes to it in the main repo will easily be forgotten here.

Reply:
I figured it's not so different to locking to a specific revision, and hopefully CI will pickup any significant issues. 🤷‍♂️ I don't have a strong opinion and am happy to change it back, and also update basic-web server to be the same if you would like.

Reply:
Updating is a lot easier with the git rev and it's also nice to have less code in the repo, so I do prefer that approach.

Reply:
@Anton-4 I started making the change as you requested to convert back to use roc_std from the roc-lang/roc repository, and it's not a simple change. The issue I see is that the current implementation of glue generates the roc_app crate which depends on roc_std. Two potential options, change glue so that it doesn't generate roc_std and instead links back to the roc repository, or manually update the generated glue files so they also use the roc-lang/roc repo version. 

Regarding your comment about having less code, I totally agree and would like to gitignore the generated code but for now there is too much that needs to be manually fixed up. I guess in that vein we could also manually update the roc_app dependency to point to the repository.

I think it is ok to leave it now using the generated glue version, and work towards fixing up glue so that we can eliminate the code altogether from the platforms.

Reply:
Sounds good, thank for investigating :)